clcache/storage.py

```diff
@@ -156,7 +156,7 @@ def clean(self, stats, maximumSize):
                                 maximumSize)
 
 
-class CacheFileWithMemcacheFallbackStrategy(object):
```

Comment:
wasn't worth keeping it, given you've disabled the warning in the other commit anyway? I guess apart from one line - the whole codebase was is still compatible with python 2.7 - but I guess on Windows it's never the case that there needs to be python 2.7 ?